applet/src/menu.c

```diff
@@ -637,9 +571,7 @@ void create_menu_entry_addl_functions_screen(void) {
                           md380_menu_entry_back+1, 0x98, 0 , 1);
   create_menu_entry_hook( *md380_menu_id + 4, wt_promtg,      create_menu_entry_promtg_screen+1,
                           md380_menu_entry_back+1, 0x98, 0 , 1);
-  create_menu_entry_hook( *md380_menu_id + 5, wt_edit,        create_menu_entry_edit_screen+1,
-                          md380_menu_entry_back+1, 0x8a, 0 , 1);
-  create_menu_entry_hook( *md380_menu_id + 6, wt_edit_dmr_id, create_menu_entry_edit_dmr_id_screen+1,
+  create_menu_entry_hook( *md380_menu_id + 5, wt_edit_dmr_id, create_menu_entry_edit_dmr_id_screen+1,
                           md380_menu_entry_back+1, 0x8a, 0 , 1);
 
  for(i=0;i<7;i++) {  // not yet known ;)
```

Comment:
Info: this must be also changed ... Number of menu entry. 

This is an nearly 1:1 copy from the bin file. there are to many manually values yet. this  have to be changed!